dds/DCPS/WaitSet.cpp

```diff
@@ -72,13 +72,17 @@ ReturnCode_t WaitSet::detach_conditions(const ConditionSeq& conds)
 {
   ACE_GUARD_RETURN(ACE_Recursive_Thread_Mutex, g, lock_,
                    RETCODE_OUT_OF_RESOURCES);
+  try {
```

Comment:
Not sure I agree with this one.  I'd rather not change code just to make Coverity happy -- there's no way to get an exception here.